src/EditMemo.js

```diff
@@ -24,8 +27,8 @@ export default function EditMemo({ memo, onSave, onDelete }) {
     <div class="container">
       <h1>メモ詳細</h1>
       <Input value={answer} handleAnswerChange={handleAnswerChange} />
-      <button onClick={handleSave}>save</button>
-      <button onClick={handleDelete}>delete</button>
+      {isLoggedIn && <button onClick={handleSave}>save</button>}
```

Comment:
[imo]
このようにしてもいいかもですね！

```suggestion
{isLoggedIn && (
  <>
    <button onClick={handleSave}>save</button>
    <button onClick={handleDelete}>delete</button>
  </>
)}
```

Reply:
[🍀 refactor: 不要な分岐をまとめた](https://github.com/reckyy/create-react-app/pull/2/commits/2d2815effbac71b29e7b179c0a45a82fa03e408b)